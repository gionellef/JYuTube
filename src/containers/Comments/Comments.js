import React from 'react';

import { CommentsHeader } from './CommentsHeader/CommentsHeader';

import './Comments.scss';

export class Comments extends React.Component {
  render() {
    return (
      <div>
        <CommentsHeader numberOfComments={this.props.numberOfComments} />
      </div>
    );
  }
}
