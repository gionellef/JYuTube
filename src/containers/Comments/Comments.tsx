import * as React from 'react';

import CommentsHeader from './CommentsHeader/CommentsHeader';
import { AddComment } from './AddComment/AddComment';
import { Comment } from './Comment/Comment';

import './Comments.scss';

interface Props {
  className?: string;
  numberOfComments?: number;
}

export class Comments extends React.Component<Props> {
  render() {
    return (
      <div>
        <CommentsHeader numberOfComments={this.props.numberOfComments} />
        <AddComment />
        <Comment />
        <Comment />
        <Comment />
      </div>
    );
  }
}
