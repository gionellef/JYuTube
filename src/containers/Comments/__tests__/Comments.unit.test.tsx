import * as React from 'react';

import { shallow } from 'enzyme';

import { Comments } from '../Comments';

describe('Comments', () => {

  test('renders without numberOfComments', () => {
    const wrapper = shallow(
      <Comments numberOfComments={86923} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});