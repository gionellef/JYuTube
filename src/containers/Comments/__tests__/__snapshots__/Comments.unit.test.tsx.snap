// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments renders without numberOfComments 1`] = `
<div>
  <CommentsHeader
    numberOfComments={86923}
  />
  <AddComment />
  <Comment />
  <Comment />
  <Comment />
</div>
`;
